spacy_llm/models/rest/base.py

```diff
@@ -17,18 +16,27 @@ def has(cls, item: int):
         return item in set(item.value for item in cls)
 
 
-class Backend(abc.ABC):
+class Model(abc.ABC):
     """Queries LLMs via their REST APIs."""
 
+    DEFAULT_STRICT = True
+    DEFAULT_MAX_TRIES = 5
+    DEFAULT_INTERVAL = 1.0
+    DEFAULT_MAX_REQUEST_TIME = 30
```

Comment:
Having this in ALL_CAPS would signal to me that they are invariant, but they are actually variables on the class that can be overwritten by subclasses, right? So basically they are invariant per subclass, but can differ between classes?

Reply:
It's not something I'd expect to happen, but theoretically yes (still disappointed there is no `const` in Python). Do you prefer them to be lowercase?

Reply:
Hm meh, I don't actually know, it's this in-between case. Maybe we just leave as-is.